### DISCUSSION TITLE
Add JobGroup to Job Parameters

### DISCUSSION THREAD

**Discussion by arrabito** [DIRACGrid/DIRAC#7597] [Q&A] [answered]
Dear all,

I've activated the use of ES for Job Parameters and it works well. Also the MemoryUsed parameter now seems correct.

However, when building histograms with OpenSearch dashboard I've realized that it would be really useful to have an additional parameter, i.e. JobGroup.

Indeed, it would be useful to filter jobs by JobGroup, since this is related to the TransformationID.

It seems to me that it's just needed to add a line in the JobWrapper.py.

If you agree, I can make a PR.

**Comment by fstagni** [comment]
Sure, go ahead with the PR.

What histograms are you creating?

**Comment by arrabito** [comment]
ok perfect.

Here an example of an histogram of Memory Used. I would like to add a Filter on a given JobGroup (i.e. on a given Transformation). Indeed "JobType" is too general for us.

![Screenshot 2024-04-26 at 11-03-41 LA MemoryUsed Histogram DL0_Reprocessing - OpenSearch Dashboards](https://github.com/DIRACGrid/DIRAC/assets/6996265/08109cae-3d86-4896-ae3a-2ecde44a4f18)

**Comment by arrabito** [comment, marked as answer]
Just created a PR:
https://github.com/DIRACGrid/DIRAC/pull/7599
it seems that a label should be added, but it seems to me that I don't have the permissions to do it.
